website/docs/r/kms_key_policy.html.markdown

```diff
@@ -0,0 +1,89 @@
+---
+subcategory: "Key Management Service"
+layout: "ibm"
+page_title: "IBM : kms-key-policies"
+description: |-
+  Manages key policies for IBM hs-crypto and kms.
+---
+
+# ibm\_kms_key_policies
+
+Provides a resource to manage key policies for hs-crypto and key-protect services. This allows key policies to be created. Key policies can be created for an existing kms key resource.
+
+
+## Example usage to create a Key and associate a key policy.
+
+```terraform
+resource "ibm_resource_instance" "kms_instance" {
+  name     = "instance-name"
+  service  = "kms"
+  plan     = "tiered-pricing"
+  location = "us-south"
+}
+
+resource "ibm_kms_key" "key" {
+  instance_id = ibm_resource_instance.kp_instance.guid
+  key_name       = "key"
+  standard_key   = false
+}
+
+resource "ibm_kms_key_policies" "keyPolicy" {
+  instance_id = ibm_resource_instance.kms_instance.guid
+  key_id = ibm_kms_key.key.key_id
+     rotation {
+       interval_month = 3
+    }
+    dual_auth_delete {
+       enabled = false
+     }
+}
+```
+
+**NOTE**
+1) `terraform destroy` does not remove the policies of the Key but only clears the state file. Key Policies get deleted when the associated key resource is destroyed.
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `instance_id` - (Required, Forces new resource, string) The hs-crypto or key-protect instance guid.
+* `endpoint_type` - (Optional, Forces new resource, string) The type of the endpoint (public or private) to be used for creating keys.
```

Comment:
Follow latest guidelines of docs..
Both arguments and attributes should be in alphabatical order..

Nested arguments should have 

Nested scheme for `argument/ attribute parent name`:

This applies to both resource and datasource